recline.module

```diff
@@ -5,8 +5,8 @@
  * Integrates Recline.js visualization tools.
  */
 
-define('RECLINE_AJAX_CALL_TIMEOUT', 'recline_ajax_call_timeout');
-define('RECLINE_AJAX_CALL_DEFAULT_TIMEOUT', 500);
+// Trying to parse a big file in a browser could freeze it.
```

Comment:
Instead of use the network latency as variable to determine if the resource can be previewed this adds a file size preview limit that doe the job. This also adds a settings where you can configure this.

This update is transparent and should not affect the user in any way.